repository/src/main/java/org/apache/atlas/repository/audit/EntityAuditListenerV2.java

```diff
@@ -288,13 +351,23 @@ public void onClassificationsDeleted(AtlasEntity entity, List<AtlasClassificatio
             MetricRecorder metric = RequestContext.get().startMetricRecord("onClassificationsDeleted");
 
             FixedBufferList<EntityAuditEventV2> events = getAuditEventsList();
+            Map<AtlasEntity, List<Map<String,Object>>> entityClassifications = new HashMap<>();
```

Comment:
I see the code from line 354 - 360 is repeated in many places. Can you move this logic to a Util or one common place ?